src/Search/Search.Tests/Tests/IndexManagementTests.cs

```diff
@@ -261,6 +261,27 @@ public void CanUpdateIndexDefinition()
         }
 
         [Fact]
+        [Trait(TestTraits.AcceptanceType, TestTraits.LiveBVT)]
+        public void CanAddSynonymFieldProperty()
+        {
+            Run(() =>
+            {
+                string synonymMapName = "names"; 
+                SearchServiceClient searchClient = Data.GetSearchServiceClient();
+
+                SynonymMap synonymMap = new SynonymMap(name: synonymMapName, format: SynonymMapFormat.Solr, synonyms: "hotel,motel");
+                searchClient.SynonymMaps.Create(synonymMap);
+
+                Index index = CreateTestIndex();
+                index.Fields.First(f => f.Name == "hotelName").SynonymMaps = new String[1] { synonymMapName };
```

Comment:
`String` is typically not capitalized in C#. You don't need the array size (it is inferred automatically). In fact, you don't even need the type:

`= new[] { synonymMapName };`